### DISCUSSION TITLE
Removing all markup?

### DISCUSSION THREAD

**Discussion by kerim** [Fevol/obsidian-criticmarkup#19] [General]
Currently, what is the easiest way to remove all CriticMarkup from a document, or to create a copy of the file without the CriticMarkup, if I want to clean the document before export? Thanks.

**Comment by Fevol** [comment]
The easiest solution at the moment are to use the `Accept all suggestions` or `Reject all suggestions` commands. As discussed in #17, this only affects Additions/Deletions/Substitutions at the moment. 

You could also copy (Mod/Ctrl+C) the note contents to a different note; by default, the plugin removes *all* markup (that is, `++}`/`{--`/... brackets) while keeping the text. This behaviour can be toggled off/on in `Editor > Remove syntax on copy`.

Of course, I'm not a big fan of these options that are available currently. For future releases, I will try to give more (and better documented) export options. Proposals for these are always welcome.

**Comment by kerim** [comment]
Thanks. Since I mostly use comments on my own documents (not suggestions), it sounds like copy and paste is the best solution right now. I suppose a slightly better workflow would be to have a "save clean copy" command that effectively did the same thing.
